mrp_project_link/wizard/create_project_mrp_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="create_project_mrp_form_view">
+            <field name="name">create.project.mrp.wiz.form</field>
+            <field name="model">create.project.mrp.wiz</field>
+            <field name="arch" type="xml">
+                <form string="Create Project">
+                    <h3>
+                    <label string="A project must be assigned in order to record the manufacturing costs."/>
+                    <label string="Do you want to create one automatically?"/>
+                    </h3>
+                    <footer>
+                        <button class="oe_highlight" 
+                                name="create_project"
+                                string="Create"
+                                type="object" />
+                        or
+                        <button class="oe_link" 
```

Comment:
Si tienen costes estimados o reales, no pueden continuar sin proyecto, o en otro caso, hay que tocar ambos módulos para verificar si la OF tiene poryecto o no, y si no tiene no hacer nada. No generar líneas de analítica.